todo_app/data/todo_items.py

```diff
@@ -0,0 +1,131 @@
+            if item.idList == self._lists['ToDo']:
+                items.append(item)
+        return items
+
+    @property
+    def doing(self):
+        items = []
+        for item in self._items:
+            if item.idList == self._lists['Doing']:
+                items.append(item)
+        return items
+
+    @property
+    def done(self):
+        items = []
+        for item in self._items:
+            if item.idList == self._lists['Done']:
+                items.append(item)
+        return items
+
+    @property
+    def done_today(self):
+        items = []
+        for item in self._items:
+            modified_date = str(item.modified)
+            completed_date = modified_date.split(" ")
+            todays_date = datetime.date.today()
+            if item.idList == self._lists['Done'] and completed_date[0] == str(todays_date):
+                items.append(item)
+        return items
+
+    @property
+    def done_before_today(self):
+        items = []
+        for item in self._items:
+            modified_date = str(item.modified)
+            completed_date = modified_date.split(" ")
+            todays_date = datetime.date.today()
+            if item.idList == self._lists['Done'] and completed_date[0] != str(todays_date):
+                items.append(item)
+        return items
+
+def get_mongodb_url():
+    mongodb_url = os.environ.get('MONGO_DB_CONNECTION')
+    return mongodb_url
+
+def connect_mongodb():
+    mongodb_url = get_mongodb_url()
+    mongo_client = pymongo.MongoClient(mongodb_url)
+    db_name = os.environ.get('MONGO_DB_NAME')
+    db = mongo_client[db_name]
+    return db
+
+def get_todo_cards():
+    db = connect_mongodb()
+    collections = db.list_collection_names()
+    
+    all_cards = []
+    for c in collections:
+        collection = db[c]
+        for card in collection.find({}):
+            all_cards.append(ToDoCard(card['_id'], card['name'], card['idList'], card['due'], card['description'], card['modified']))
+            
+    return all_cards
+
+def create_todo_card(new_cardtitle, new_carddesc):
+    db = connect_mongodb()
+    new_card = ToDoCard(None, new_cardtitle, "ToDo", datetime.datetime.today(), new_carddesc, datetime.datetime.today())
+    card = new_card.get_card_as_dictionary()
+    db['ToDo'].insert_one(card)
+
+def move_todo_card(card_id, new_list_id):
+    db = connect_mongodb()
+    collections = db.list_collection_names()
+
+    for c in collections:
+        collection = db[c]
+        for card in collection.find({}): 
+            if str(card['_id']) == str(card_id):
+                new_card = ToDoCard(0, card['name'], new_list_id, card['due'], card['description'], datetime.datetime.today())
+                new_collection = db[new_list_id]
+                new_collection.insert_one(new_card.get_card_as_dictionary())
+                result = collection.delete_one({'_id' : card['_id']})
+                print(result)
+                break
```

Comment:
Rather than looping over all of your cards, this would make more sense finding just the relevant card, e.g. 

```suggestion
for c in collections:
        collection = db[c]
        card = collection.find_one({"_id": card_id})
        new_card = ToDoCard(0, card['name'], new_list_id, card['due'], card['description'], datetime.datetime.today())
        new_collection = db[new_list_id]
        new_collection.insert_one(new_card.get_card_as_dictionary())
        result = collection.delete_one({'_id' : card['_id']})
        print(result)
        break
```